spec/features/request_ride_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+feature "Request ride" do
```

Comment:
Take a look at https://www.relishapp.com/rspec/rspec-rails/docs/feature-specs/feature-spec. A common way to create features specs is to use the keyword scenario to describe the scenario the user is going through. Here's another resource that is helpful in DRYing up your feature specs. https://robots.thoughtbot.com/rspec-integration-tests-with-capybara